### PR TITLE
Add DC Motor Code

### PR DESCRIPTION
**Pull request by aditinikam**

### CODE REVIEW

File: Arduino/Actuator/Aditi Nikam/dcmotor.ino [hunk 1]
```diff
@@ -0,0 +1,41 @@
+int motorPin = 2;
+int input1 = 8;
+int input2 = 9;
+void setup() {
+  Serial.begin(9600);
+  pinMode(motorPin,OUTPUT);
+  pinMode(input1,OUTPUT);
```
Comment (aditya-c137) [inline Arduino/Actuator/Aditi Nikam/dcmotor.ino:7]: Where are these input pins connected?

**Comment by aditya-c137** [comment]
Write code for solenoid valve next

File: Arduino/Actuator/Aditi Nikam/dcmotor.ino (hunk 1 above)
Comment (aditinikam) [reply Arduino/Actuator/Aditi Nikam/dcmotor.ino:7]: 2, 8 and 9 digital pin

Comment (aditya-c137) [reply Arduino/Actuator/Aditi Nikam/dcmotor.ino:7]: I meant what is connected to pin numbers 8 and 9... sorry the question wasn't clear 😛

Comment (aditinikam) [reply Arduino/Actuator/Aditi Nikam/dcmotor.ino:7]: Ooh sorry! 8 and 9 are connected to input pins which decides the direction of the motor and 2 is connected for the speed.

Comment (aditya-c137) [reply Arduino/Actuator/Aditi Nikam/dcmotor.ino:7]: Ohkkk cool... Noice

**Pull request merged**